source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,337 @@ Sorts Builders
+by how closely they match your search string. Each of your search results has a
+:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>`, a
+numerical value indicating how well that result matches your search. 
+Use the ``Sorts.metaTextScore()`` static factory method to build your sort
+criteria to sort by the text score.
+
+.. warning:: Make Sure to Create a Text Index
+
+   Text searches require a :manual:`text index </core/index-text/>` to locate
+   and score your results. See the server manual documentation for more
+   information on how to
+   :manual:`create a text index </core/index-text/#create-text-index>`.
+
+.. example::
+
+   In the following code example, we show how you can use the
+   ``Sorts.metaTextScore()`` method to sort the results of a text
+   search on the :ref:`sort_example collection <sorts-builders-sort-example>`.
+   The code example uses the :doc:`Filters </fundamentals/builders/filters>`,
+   :doc:`Indexes </fundamentals/builders/indexes>`, and
+   :doc:`Projections </fundamentals/builders/projections>` builders.
+   The code example performs the following actions:
+
+   #. Creates a text index for your
+      :ref:`sort_example collection <sorts-builders-sort-example>`
+      on the ``food`` field.
+   #. Runs your text search for the phrase "maple donut".
+   #. Projects text scores into your query results as the
+      ``score`` field. This projection is optional if your MongoDB instance is
+      running MongoDB 4.4 or later. 
+   #. Sorts your results by text score (best match first).
+
+   .. code-block:: java
+   
+      import com.mongodb.client.model.Sorts;
+      import com.mongodb.client.model.Projections;
+      import com.mongodb.client.model.Filters;
+      import com.mongodb.client.model.Indexes;
+
+      // <MongoCollection setup code here>
+      
+      collection.createIndex(Indexes.text("food"));
+      Bson metaTextScoreSort = Sorts.metaTextScore("score");
+      Bson metaTextScoreProj = Projections.metaTextScore("score");
+      String searchTerm = "maple donut";
+      Bson searchQuery = Filters.text(searchTerm);
+      collection.find(searchQuery)
+              .projection(metaTextScoreProj)
+              .sort(metaTextScoreSort)
+              .into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+   
+   The output of the code snippet above should look something like this: 
+   
+   .. code-block:: json
+
+      {"_id": 6, "letter": "c", "food": "maple donut", "score": 1.5}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee", "score": 0.75}
+      {"_id": 3, "letter": "a", "food": "maple syrup", "score": 0.75}
+
+.. note:: MongoDB 4.4 ``$meta`` Behavior
+
+  When using MongoDB 4.4 or later, projecting ``Projections.metaTextScore()`` 
+  into your ``FindIterable`` instance is not necessary to sort on
+  ``Sorts.metaTextScore()``. In addition, the field name you specify in a
```

Comment:
Did you mean "to sort on the text score"? It doesn't make sense to "sort on [the metaTextScore method]".

Reply:
Thank you for the note. On rereading, I agree this is incorrect. Fix: https://github.com/mongodb/docs-java/pull/59/commits/56e21a4cdbae14d49a18678a1ff7401e223daa6e